spec/queries/reports/list_query_spec.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+RSpec.describe Reports::ListQuery do
+  describe ".call" do
+    fab!(:current_user) { Fabricate(:admin) }
+
+    subject(:result) { described_class.call }
```

Comment:
`subject` should be the first statement of a block, weird the linter didn’t get it :thinking:

Reply:
Ok will fix :+1: